docs/static/breaking-changes.asciidoc

```diff
@@ -34,7 +34,9 @@ These changes can impact any instance of Logstash and are plugin agnostic, but o
 * Configurations provided with `-f` or `config.path` will not be appended with `stdin` input and `stdout` output automatically.
 
 [float]
+=== Plugin Changes
 
+[float]
 ==== Elasticsearch output changes
 
 * The default `document_type` has changed from `logs` to `doc`.
```

Comment:
Let's include some next-step information. For example, referencing why this change was made (document type deprecated in ES 6.0) and include a link for further information (link to Elasticsearch's own docs on this topic).

Reply:
For changing the document_type from `logs` to `doc`, does it make it sense to use the same wording from the changelog of the plugin - ```Change default document type to 'doc' from 'logs' to align with beats and reflect the generic nature of logstash```

There is already a link to the Elasticsearch docs on removal of types - is there another link you are looking to include here?